.goreleaser.yml

```diff
@@ -55,10 +55,6 @@ release:
   extra_files:
     - glob: 'terraform-registry-manifest.json'
       name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
-  # If you want to manually examine the release before its live, uncomment this line:
-  # draft: true
-  prerelease: auto
-  make_latest: '{{ not .Prerelease }}'
```

Comment:
default values are `prerelease: false` and `make_latest: true`
https://goreleaser.com/customization/release/#github